bin/install

```diff
@@ -2,6 +2,9 @@
 set -eu
 [ "${BASH_VERSINFO[0]}" -ge 3 ] && set -o pipefail
 
+BASE_URL="${BASE_URL:-https://storage.googleapis.com/golang}"
```

Comment:
I don't see any reason why we should continue to use this URL.

The official https://golang.org/dl/ page is now using `dl.google.com`. Let's default to that value instead.

Reply:
Thanks for pointing that out. Updated.